container/nginx/confs/proxy.tlsconf

```diff
@@ -37,6 +38,12 @@ server {
 
     resolver    8.8.8.8 1.1.1.1 valid=600s;
 
+    # Add Alt-Svc header to negotiate HTTP/3.
+    add_header alt-svc 'h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400';
```

Comment:
The add_header directives will be lost. Sadly they need to be duplicated wherever we want to return them

> [There could be several add_header directives. These directives are inherited from the previous configuration level if and only if there are no add_header directives defined on the current level.](http://nginx.org/en/docs/http/ngx_http_headers_module.html#add_header)

Reply:
Alright I see `There could be several add_header directives. These directives are inherited from the previous configuration level if and only if there are no add_header directives defined on the current level.`